core/templates/pages/exploration-player-page/layout-directives/audio-bar.directive.html

```diff
@@ -27,8 +27,11 @@
           <md-progress-linear md-mode="indeterminate"></md-progress-linear>
         </div>
         <div ng-if="!audioLoadingIndicatorIsShown">
-          <md-slider ng-model="track.progress" ng-model-options="{ getterSetter: true }" aria-label="audio-slider">
-          </md-slider>
+          <oppia-audio-slider [value]="AudioPlayerService.getCurrentTime()"
+                              [max]="AudioPlayerService.getAudioDuration()"
```

Comment:
Instead of attaching the `AudioPlayerService` here can we have precomputed currentTime and audioDuration var here?

Reply:
The view only updates when we call $scope.$applyAsync(); (I checked it by putting breakpoints). I thought this would make a lot of function calls but it doesn't. (My initial try was to use this and not use $applyAsync, but the view didn't update. So I checked by putting breakpoints, and it only calls when a digest cycle is run).